### DISCUSSION TITLE
Missed the MOK secureboot enrollment prompt - how to re-initiate

### DISCUSSION THREAD

**Discussion by citadel-lewis** [linux-surface/linux-surface#940] [General]
I installed linux-surface-secureboot and wrote down the password for enrollment at next boot. When I rebooted, I realised my bluetooth keyboard wasn't connected and by the time I connected a USB keyboard the prompt to enter the enrollment password was gone. I tried to re-install linux-surface-secureboot to reinitiate the process but it didn't work. Any idea on how I can complete the secureboot enrollment process?

**Comment by qzed** [comment]
You can uninstall and then reinstall the MOK package. This will trigger the prompt again.

**Reply by citadel-lewis** [reply]
I was able to trigger the prompt again by issuing this command:
`mokutil --import /usr/share/linux-surface-secureboot/surface.cer`
and then entering a password
